webrtc/udp_transport.nim

```diff
@@ -47,16 +49,18 @@ proc new*(T: type UdpTransport, laddr: TransportAddress): T =
 
   self.dataRecv = newAsyncQueue[UdpPacketInfo]()
   self.udp = newDatagramTransport(onReceive, local = laddr)
+  trackCounter(UdpTransportTrackerName)
   return self
 
-proc close*(self: UdpTransport) =
+proc stop*(self: UdpTransport) {.async: (raises: []).} =
```

Comment:
Doesn't `UdpTransport` have a start proc?

Reply:
Hum... no it doesn't. I made it start in `new`.
But I don't like `close` because we didn't open anything.
And I don't like the solution with both a start and a new because it just means splitting new in two proc just for like 2 loc.

In my opinion, you stop a Transport from accepting / connecting and you close a Connection. It makes more sense to me.

Reply:
>  I made it start in new.

I'm not sure this is a good idea.

Reply:
> But I don't like close because we didn't open anything.

Using the same logic, we haven't started anything either.

Reply:
I switched back to close. DatagramTransport uses close aswell. I'm not entirely satisfied by this but both solutions annoy me.

Reply:
My point was more about the lack of an explicit start.